portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro

```diff
@@ -866,6 +871,21 @@ definition {
 		}
 	}
 
+	macro sessionPause {
+		if (isSet(time)) {
+
+			// Pause for reusable purpose
+
+			Pause(locator1 = "${time}");
+		}
+		else if (!(isSet(time))) {
+
+			// It needs a 30 second pause which is the time required for the command to be executed. For more information about pause see LRAC-9518
+
+			Pause(locator1 = "30000");
+		}
+	}
+
```

Comment:
Hey @calvinctvo, I don't think we're going to need this macro. What do you think?

Reply:
Hey @ViniLopes20 , I create this macro for closeAllSessionsAndWait macro. Since this particular pause will wait for 30s to close all sessions.